src/collective/importexport/browser/import_view.py

```diff
@@ -258,16 +274,61 @@ def export_file(result, header_mapping, request=None):
                     continue
                 if value is field.missing_value:
                     continue
+                if IRichTextValue.providedBy(value):
+                    # Convert to plain text
+                    transforms = api.portal.get_tool('portal_transforms')
+                    value = transforms.convertTo(
+                                target_mimetype='text/plain',
+                                orig=value.raw_encoded,
+                                encoding=value.encoding,
+                                mimetype=value.mimeType)
+                    value = safe_unicode(value.getData())
+                    break
+
+                if IList.providedBy(field):
+                    # Check if items in list point to objects and use titles
+                    # instead
+                    names=[]
+                    for v in value:
+                        # Loop through choices
+                        query = dict(id=v)
+                        catalog = api.portal.get_tool("portal_catalog")
```

Comment:
Getting a tool takes some time, so it would be better if you move this out of the for loop, no need to get it multiple times.